src/commands/flex/plugins/remove.js

```diff
@@ -16,6 +16,6 @@ class FlexPluginsRemove extends FlexPlugin {
   }
 }
 
-FlexPluginsRemove.description = createDescription('Removes your Flex plugin', true);
+FlexPluginsRemove.description = createDescription('Removes the Flex plugin', true);
```

Comment:
or should this one be `Removes Flex plugin`? just want to be consistent. personally like the `the` but @rnairtwilio can decide